kube_scheduler/pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
     "Private :: Do Not Upload",
 ]
 dependencies = [
-    "datadog-checks-base>=25.1.0",
+    "datadog-checks-base[deps,kube]>=25.1.0",
```

Comment:
Should instead use https://github.com/DataDog/integrations-core/blob/c73364ffb6c7c31cbfbe231e943bc73f0db0baf6/datadog_checks_dev/datadog_checks/dev/plugin/hatch/environment_collector.py#L82